packages/bot-web-ui/src/pages/server-side-bot/bot-list/common/login-modal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Dialog, Icon, Text } from '@deriv/components';
+import { getLanguage, localize } from '@deriv/translations';
+import { observer } from '@deriv/stores';
+import { redirectToLogin, redirectToSignUp } from '@deriv/shared';
+import './login-modal.scss';
+
+type TLoginModal = {
+    is_login_modal_visible: boolean;
+    setLoginModalVisble: (value: boolean) => void;
+};
+
+const loginModal = observer(({ is_login_modal_visible, setLoginModalVisble }: TLoginModal) => {
```

Comment:
```suggestion
const LoginModal = observer(({ is_login_modal_visible, setLoginModalVisble }: TLoginModal) => {
```
it is sort of standard for components to start with capital letters